### PR TITLE
[feature] Added downloading Actions instructions to README

### PR DESCRIPTION
**Pull request by 5HT2**
Because you **know** users are too dumb to figure it out

### CODE REVIEW

**Comment by JerryWester** [comment]
Smh, including a backdoor in the code

**Comment by 5HT2** [comment]
Please notice the very professional image ;))

**Comment by JerryWester** [comment]
Ah, yes, very professional GIMP skillz

File: README.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ The Fabric 1.16.3 version is in active development.
 
 ## Installing
 
-KAMI is a fabric mod. Download the [latest release](https://github.com/zeroeightysix/KAMI/releases), or preferably, [**build it yourself**](#building).
+KAMI is a fabric mod. Download the [latest release](#downloading), or [build it yourself](#building).
```
Comment (zeroeightysix) [inline README.md]: Why remove the `preferably`?

File: README.md [hunk 2]
```diff
@@ -37,6 +37,21 @@ If you encounter bugs or crashes, please [open a new issue](../../issues/new/cho
 
 To ask for help, join the [Discord](http://discord.gg/9hvwgeg).
 
+## Downloading
+
+1. Go to the [Actions](https://github.com/zeroeightysix/KAMI/actions) tab
+2. Click on the latest action
+3. Click on "KAMI mod jar"
+4. Extract downloaded zip file
+5. [Install](#installing)
```
Comment (zeroeightysix) [inline README.md:48]: IIRC you need to be logged in to download an artifact. Mention this?

**Review by zeroeightysix — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (5HT2) [reply README.md]: because actions builds the latest commit for you, and it's a consistent build environment

File: README.md (hunk 2 above)
Comment (5HT2) [reply README.md:48]: I didn't know that. I'll fix it once I'm on a laptop

**Review by zeroeightysix — APPROVED** [review]

**Comment by zeroeightysix** [comment]
Thanks!

**Pull request merged**